terraform/provider/Makefile

```diff
@@ -1,6 +1,6 @@
 
 test:
-	TF_ACC=yes HAPPY_API_BASE_URL=https://fake.happy-api.io go test -cover -v ./... -run ^$(name)
+	go test -cover -v ./... -run ^$(name)
```

Comment:
moved this into the tests so they are easier to run. I wasn't a fan of magic environment variables (magic to me at least)